### PR TITLE
v2.13.0

### LINKED ISSUE #386: getMinMaxLOD calculation
Please look at the getMinMaxLOD calculation in offline-tiles-advanced-src.js.   I believe that line 241 should be a subtract not an add.

var min = map.getLevel() + minZoomAdjust;

Great library by the way.  It is turning out to be very useful.

### LINKED ISSUE #390: In offlineFeatureManager, wait for pushPhantomGraphic callback before setting attributes and styles
When extending a layer, on the private _pushValidated(edit)FeaturetoDB methods, it's possible to try to try to set the phantom graphic attribute and style before the _editStore.pushPhantomGraphic callback is complete.  This would result in a "cannot set attribute of null" error and the method's deferred will not resolve.

These two lines should be moved up within the pushPhantomGraphic callback function so they are only run with a successful callback:
`                                        domAttr.set(phantomUpdate.getNode(), "stroke-dasharray", "5,2");
                                        domStyle.set(phantomUpdate.getNode(), "pointer-events", "none");`

### PR DESCRIPTION
**Pull request by robkspeer**
## Version 2.13.0 - Aug. 10, 2015

##Bug Fixes**
* Closes #386 - getMinMaxLOD calculation
* Closes #390 - In offlineFeatureManager, wait for pushPhantomGraphic
callback before setting attributes and styles of phantom graphics.
* For token based security, when syncing feature edits, provide direct
access to the featureLayer.credentials.token instead of looping thru the
kernel object (as introduced in v2.12.0)

### CODE REVIEW

File: lib/edit/offlineFeaturesManager.js [hunk 1]
```diff
@@ -2097,7 +2101,7 @@ define([
                  * @returns {*|r}
                  * @private
                  */
-                _makeEditRequest: function(url,adds, updates, deletes, callback, errback) {
+                _makeEditRequest: function(layer,adds, updates, deletes, callback, errback) {
```
Comment (andygup) [inline lib/edit/offlineFeaturesManager.js]: Minor note: Can you also make sure the documentation [param](https://github.com/Esri/offline-editor-js/blob/master/lib/edit/offlineFeaturesManager.js#L2093) is updated?

**Comment by robkspeer** [comment]
originally i didn't want to mess with the input parameters to the _makeEditRequest function, so i used kernel to access tokens at the root level.  that required looping thru every credential and trying to line up the server the token was issued from with the server the feature is hosted on.  but by passing the entire layer into that function (and not just the layer.url) we're able to directly access the layer.credentials.token for that feature without messing with the looping and matching.

so... yes, I regret to say I removed kernel again after having just adding it.

**Comment by andygup** [comment]
I suspected that might be the case...Do you wanna go ahead and update gh-pages and create a new release point?

**Comment by andygup** [comment]
Hey Rob,

Are you able to update gh-pages this morning and create a release point? This last manual step should go complete away once I get the shell script bolted into place.

By updating gh-pages folks can access the latest library files via the github cdn, for example: http://esri.github.io/offline-editor-js/dist/offline-edit-src.js

Andy

**Pull request merged**